src/test/java/org/exist/xquery/modules/jfreechart/ExampleModuleTest.java

```diff
@@ -32,12 +12,12 @@
 
 
 
-    private Sequence executeQuery(final String xquery) throws EXistException, PermissionDeniedException, XPathException {
-        final BrokerPool pool = existEmbeddedServer.getBrokerPool();
-        final XQuery xqueryService = pool.getXQueryService();
-
-        try(final DBBroker broker = pool.get(Optional.of(pool.getSecurityManager().getSystemSubject()))) {
-            return xqueryService.execute(broker, xquery, null);
-        }
-    }
+//    private Sequence executeQuery(final String xquery) throws EXistException, PermissionDeniedException, XPathException {
```

Comment:
Tests seem to be commented out?

Reply:
yea nothing happened here. I will remove the test completely.